.chloggen/codeboten_beta-prom-normalize.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: prometheusreceiver
```

Comment:
That is also relevant to Prometheus exporters? (tests were adjusted exporter/prometheusexporter/collector_test.go)The Prometheus and Prometheus remote write exporters will be producing different names for exporter series now? Shall we provide some migration instructions for users like how to keep the old behavior, etc.?

Reply:
I've updated the changelog to include more details about this.